content/en/error_tracking/frontend/mobile/reactnative.md

```diff
@@ -5,4 +5,914 @@ code_lang: reactnative
+</div>
+
+You can track events such as crashes and network requests when your application is in the background (for example, when no active view is available).
+
+Add the following snippet during initialization in your Datadog configuration:
+
+```javascript
+configuration.trackBackgroundEvents = true;
+```
+{{% /collapse-content %}}
+
+
+{{% collapse-content title="Send data when device is offline" level="h4" %}}
+
+The React Native SDK ensures availability of data when your user device is offline. In cases of low-network areas, or when the device battery is too low, all events are first stored on the local device in batches. They are sent as soon as the network is available, and the battery is high enough to ensure the React Native SDK does not impact the end user's experience. If the network is not available with your application running in the foreground, or if an upload of data fails, the batch is kept until it can be sent successfully.
+
+This means that even if users open your application while offline, no data is lost.
+
+**Note**: The data on the disk is automatically deleted if it gets too old to ensure the React Native SDK does not use too much disk space.
+
+{{% /collapse-content %}}
+
+
+{{% collapse-content title="Data storage" level="h4" %}}
+##### Android
+
+Before data is uploaded to Datadog, it is stored in cleartext in your application's cache directory. This cache folder is protected by [Android's Application Sandbox][11], meaning that on most devices this data can't be read by other applications. However, if the mobile device is rooted, or someone tampers with the Linux kernel, the stored data might become readable.
+
+##### iOS
+
+Before data is uploaded to Datadog, it is stored in cleartext in the cache directory (`Library/Caches`) of your [application sandbox][12], which can't be read by any other app installed on the device.
+
+{{% /collapse-content %}}
+
+
+{{% collapse-content title="Development mode" level="h4" %}}
+While in development mode, your application can submit extra events related to the React Native tooling, such as code transformation errors and requests to a local development server.
+
+To prevent these events from showing in the dashboard, you can disable errors and resources tracking in dev mode using the `__DEV__` flag:
+
+```js
+const config = new DatadogProviderConfiguration(
+	CLIENT_TOKEN,
+	ENVIRONMENT,
+	APPLICATION_ID,
+	true,
+	!__DEV__  /* trackResources will be false in DEV mode, true otherwise */,
+	!__DEV__  /* trackErrors will be false in DEV mode, true otherwise */,
+	trackingConsent
+)
+```
+{{% /collapse-content %}}
+
+
+{{% collapse-content title="New architecture support" level="h4" %}}
+The [React Native new architecture][601] is supported by the React Native SDK in version `>=1.8.0`.
+
+The minimum supported React Native version for the new architecture is `0.71`.
+
+[601]: https://source.android.com/security/app-sandbox
```

Comment:
It looks like this might be the link intended for reference link `[12]` on line `453`, but not sure if this is the expected page for `React Native new architecture`. Could you please confirm?